[2025-07-16T14:18:35.316+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: banking_data_quality_dag.risk_based_check manual__2025-07-16T14:18:30+00:00 [queued]>
[2025-07-16T14:18:35.328+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: banking_data_quality_dag.risk_based_check manual__2025-07-16T14:18:30+00:00 [queued]>
[2025-07-16T14:18:35.329+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-07-16T14:18:35.345+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): risk_based_check> on 2025-07-16 14:18:30+00:00
[2025-07-16T14:18:35.350+0000] {standard_task_runner.py:57} INFO - Started process 333 to run task
[2025-07-16T14:18:35.353+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'banking_data_quality_dag', 'risk_based_check', 'manual__2025-07-16T14:18:30+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/banking_dq_dag.py', '--cfg-path', '/tmp/tmpko0ijpti']
[2025-07-16T14:18:35.356+0000] {standard_task_runner.py:85} INFO - Job 13: Subtask risk_based_check
[2025-07-16T14:18:35.404+0000] {task_command.py:416} INFO - Running <TaskInstance: banking_data_quality_dag.risk_based_check manual__2025-07-16T14:18:30+00:00 [running]> on host be9bd688ef95
[2025-07-16T14:18:35.488+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='banking_data_quality_dag' AIRFLOW_CTX_TASK_ID='risk_based_check' AIRFLOW_CTX_EXECUTION_DATE='2025-07-16T14:18:30+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-16T14:18:30+00:00'
[2025-07-16T14:18:35.490+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-07-16T14:18:35.491+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/***/jobs/monitoring_audit.py']
[2025-07-16T14:18:35.501+0000] {subprocess.py:86} INFO - Output:
[2025-07-16T14:18:36.408+0000] {subprocess.py:93} INFO - ⚠️ Risky behavior detected. See logs/risk_check_log.csv and risk_tag_table.csv
[2025-07-16T14:18:36.494+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-07-16T14:18:36.519+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=banking_data_quality_dag, task_id=risk_based_check, execution_date=20250716T141830, start_date=20250716T141835, end_date=20250716T141836
[2025-07-16T14:18:36.569+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-07-16T14:18:36.588+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
